sympy/functions/elementary/tests/test_exponential.py

```diff
@@ -381,7 +381,7 @@ def test_lambertw():
     assert LambertW(2, evaluate=False).is_real
     p = Symbol('p', positive=True)
     assert LambertW(p, evaluate=False).is_real
-    assert LambertW(p - 1, evaluate=False).is_real is None
+    assert LambertW(p - 1, evaluate=False).is_real is False
```

Comment:
Using floats over symbolic expressions is generally not a good idea, as it can lead to subtle issues. For example, what if 2.71 is not enough precision for some expression. What if someone passes in `exp(1)` to the function? `(exp(1) - 2.71).is_positive` will be True, even though the intended effect will be for it to be True.